client/scss/components/_listing.scss

```diff
@@ -52,7 +52,10 @@ ul.listing {
 
   td {
     vertical-align: middle;
-    overflow-wrap: anywhere;
+
+    &.wrap-anywhere {
+      overflow-wrap: anywhere;
+    }
```

Comment:
If this is all the class does (i.e. a utility class) and we use it on the element directly in the markup, I think we should just use Tailwind

```suggestion
```